ops/terraform/services/server/certstores.tf

```diff
@@ -0,0 +1,143 @@
+locals {
+  client_certificates_map = zipmap(
+    data.aws_ssm_parameters_by_path.client_certificates.names,
+    nonsensitive(data.aws_ssm_parameters_by_path.client_certificates.values)
+  )
+  truststore_local_path = "${path.module}/generated/truststore.pfx"
+  truststore_s3_key     = "truststore.pfx"
+  truststore_certs = [
+    for ssm_path, pubkey in local.client_certificates_map
+    : {
+      alias  = element(split("/", ssm_path), length(split("/", ssm_path)) - 1)
+      pubkey = pubkey
+    }
+    if strcontains(ssm_path, "/client_certificates/")
+  ]
+
+  keystore_local_path = "${path.module}/generated/keystore.pfx"
+  keystore_s3_key     = "keystore.pfx"
+  keystore_base64     = local.sensitive_service_config["server_keystore_base64"]
+}
+
+data "aws_ssm_parameters_by_path" "client_certificates" {
+  path = "/bfd/${local.env}/${local.service}/nonsensitive/client_certificates"
+}
+
+resource "aws_s3_bucket" "certstores" {
+  bucket        = "bfd-${local.env}-${local.service}-certstores"
```

Comment:
What do you think about using `bucket_prefix` instead of `bucket` in this instance? Probably ought to be what we do going forward. I don't think adopting that here would require any coordination/orchestration beyond this module... ought to be safe.

Reply:
Yeah, that's totally fine with me. I'll make the change.

Reply:
I've made this change in [2e59e2c](https://github.com/CMSgov/beneficiary-fhir-data/pull/2575/commits/2e59e2cfb191d19562c72001588888de4813fb0c), and have re-`apply`'d `server` to `test` making sure that it `apply`s just fine.